pkg/operator/operator.go

```diff
@@ -124,6 +124,11 @@ func NewOperator() (context.Context, *Operator) {
 		GracePeriod: 5 * time.Second,
 	})
 
+	// Logging
```

Comment:
Looks good! Mind providing the difference in what you see in the terminal in the description of the PR?

Reply:
Ok, I've updated the description.